adaptive-app/src/components/ui/chat/chat-message.tsx

```
@@ -403,54 +428,103 @@ const ReasoningBlock = ({ part }: { part: ReasoningUIPart }) => {
+            </div>
+          </div>
         </div>
       );
+
     case "result":
+      if (!result) return null;
+
+      const searchData = result as {
+        query: string;
+        results: Array<{ title: string; url: string; snippet: string }>;
+      };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)